<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>0</y>
      <width>240</width>
      <height>740</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>238</width>
       <height>738</height>
      </rect>
     </property>
     <widget class="QGraphicsView" name="paletteView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>238</width>
        <height>740</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="canvasView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>736</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>960</width>
      <height>736</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCharger_zone"/>
    <addaction name="actionSauvegarder_zone"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <widget class="QMenu" name="menuPalette">
     <property name="title">
      <string>Palette</string>
     </property>
     <addaction name="actionEnvironnement"/>
     <addaction name="actionElements"/>
    </widget>
    <addaction name="menuPalette"/>
    <addaction name="actionEffacer_Tout"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutils"/>
  </widget>
  <action name="actionCharger_zone">
   <property name="text">
    <string>Charger zone</string>
   </property>
  </action>
  <action name="actionSauvegarder_zone">
   <property name="text">
    <string>Sauvegarder zone</string>
   </property>
  </action>
  <action name="actionEffacer_Tout">
   <property name="text">
    <string>Effacer Tout</string>
   </property>
  </action>
  <action name="actionEnvironnement">
   <property name="text">
    <string>Environnement</string>
   </property>
  </action>
  <action name="actionElements">
   <property name="text">
    <string>Elements</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
